python-example-elasticsearch-extension/README.md

```diff
@@ -1,70 +1,41 @@
-    XXXX-XX-XXTXX:XX:XX.XXX-XX:XX    python-example-elasticsearch-extension launching extension
-    XXXX-XX-XXTXX:XX:XX.XXX-XX:XX    [python-example-elasticsearch-extension] Registering...
-    XXXX-XX-XXTXX:XX:XX.XXX-XX:XX    [python-example-elasticsearch-extension] Registered with ID: 6ec8756c-4830-458b-9dda-156e5dda1cc1
-    XXXX-XX-XXTXX:XX:XX.XXX-XX:XX    [python-example-elasticsearch-extension] Waiting for event...
-    XXXX-XX-XXTXX:XX:XX.XXX-XX:XX    [python-example-elasticsearch-extension] Received event: {"eventType": "INVOKE", "deadlineMs": 1596756305517, "requestId": "4b61d2be-3ba0-4e99-9121-30268b462c77", "invokedFunctionArn": "", "tracing": {"type": "X-Amzn-Trace-Id", "value": ""}}
-    XXXX-XX-XXTXX:XX:XX.XXX-XX:XX    [python-example-elasticsearch-extension] Sending payload: {"functionName": "defaultFunctionName", "functionVersion": "$LATEST", "requestId": "4b61d2be-3ba0-4e99-9121-30268b462c77", "waitDuration": 3787.946044921875}
-    XXXX-XX-XXTXX:XX:XX.XXX-XX:XX    [python-example-elasticsearch-extension] Attempting POST to: https://ec2-XXX-XXX-XXX-XXX.compute-1.amazonaws.com/extensions/_doc
-    XXXX-XX-XXTXX:XX:XX.XXX-XX:XX    [python-example-elasticsearch-extension] Response: {"_index":"extensions","_type":"_doc","_id":"eW8WxnMB4bYnBqSFvK5w","_version":1,"result":"created","_shards":{"total":2,"successful":1,"failed":0},"_seq_no":0,"_primary_term":1}
-    XXXX-XX-XXTXX:XX:XX.XXX-XX:XX    [python-example-elasticsearch-extension] Waiting for event...
-    ...
-    ...
-    Function logs...
-    ...
-    ...
-    XXXX-XX-XXTXX:XX:XX.XXX-XX:XX    END RequestId: 9ca08945-de9b-46ec-adc6-3fe9ef0d2e8d
-    XXXX-XX-XXTXX:XX:XX.XXX-XX:XX    REPORT RequestId: 9ca08945-de9b-46ec-adc6-3fe9ef0d2e8d Duration: 80.36 ms Billed Duration: 100 ms Memory Size: 128 MB Max Memory Used: 67 MB Init Duration: 297.83 ms
+After invoking the function and receiving the shutdown event, you should now see log messages from the example extension written back to the extension logs. For example:
 ```
-
-
-
-
-
+...
+BATCH RECEIVED: [{'time': '2020-11-12T00:30:48.883Z', 'type': 'platform.start', 'record': {'requestId': '49ae0e5f-bc60-4521-81e3-6e41d6bcb55c', 'version': '$LATEST'}}, {'time': '2020-11-12T00:30:48.993Z', 'type': 'platform.logsSubscription', 'record': {'name': 'logs_api_http_extension.py', 'state': 'Subscribed', 'types': ['platform', 'function']}}, {'time': '2020-11-12T00:30:48.993Z', 'type': 'platform.extension', 'record': {'name': 'logs_api_http_extension.py', 'state': 'Ready', 'events': ['INVOKE', 'SHUTDOWN']}}, {'time': '2020-11-12T00:30:49.017Z', 'type': 'platform.end', 'record': {'requestId': '49ae0e5f-bc60-4521-81e3-6e41d6bcb55c'}}, {'time': '2020-11-12T00:30:49.017Z', 'type': 'platform.report', 'record': {'requestId': '49ae0e5f-bc60-4521-81e3-6e41d6bcb55c', 'metrics': {'durationMs': 15.74, 'billedDurationMs': 100, 'memorySizeMB': 128, 'maxMemoryUsedMB': 62, 'initDurationMs': 226.3}}}]
```

Comment:
You removed this output, right?

Reply:
I have updated this to be an output from this extension.